### PR TITLE
adds useful commands to tree

### PR DESCRIPTION
**Pull request by audibleblink**
This PR adds some command that were useful to me when learning about tree.

### CODE REVIEW

File: pages/common/tree.md [hunk 1]
```diff
@@ -17,3 +17,11 @@
...
 
 `tree -h`
+
+- Print the full path for each file
+
+`tree -f`
+
+- Print the tree without lines and indentation. Useful when used with -f
+
+`tree -f`
```
Comment (waldyrious) [inline pages/common/tree.md]: You apparently forgot to change the actual example :)

**Comment by audibleblink** [comment]
Rookie move. Fixed. Thanks.

**Comment by waldyrious** [comment]
lgtm :+1:

**Comment by felixonmars** [comment]
:+1:

**Comment by ostera** [comment]
@audibleblink, please rebase this branch with latest master so we can merge it in.

**Comment by audibleblink** [comment]
@ostera done

**Comment by waldyrious** [comment]
Thanks!

**Pull request merged**